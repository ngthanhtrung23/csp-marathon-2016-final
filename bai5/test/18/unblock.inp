..1112
334452
600.57
688.57
..9AA.
BB9...
