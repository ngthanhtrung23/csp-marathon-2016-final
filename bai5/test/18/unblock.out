23
1 L 1
5 U 1
7 D 1
2 D 1
8 R 2
9 U 1
B R 3
6 D 2
0 L 1
9 D 1
8 L 3
9 U 2
A L 1
B L 1
5 D 3
1 R 2
4 R 1
9 U 2
0 R 3
9 D 1
1 L 1
2 U 1
0 R 1

